docs/enhancements/tenancy-based-auth.md

```diff
@@ -0,0 +1,83 @@
+# Tenancy Based Authorization
```

Comment:
Im lacking here the configuration flow - who / how will tell AMS what's allowed, what's not.

Reply:
The configuration is already existing for the users. We are just query it

Reply:
It would be great if you could show how it's being done for the full context (screenshots? links to OCM?)

Reply:
New users are configured in configuration files in git. This is out of scope for this document. I can add the screen shot of the UI button that allows the users themselves to add collaborators for their clusters